scripts/proposals/proposal_04_service_registry_token_utility_change_drainer_gnosis.js

```diff
@@ -0,0 +1,89 @@
+/*global process*/
+
+const { ethers } = require("ethers");
+
+async function main() {
```

Comment:
A proposal to change the drainer for the ServiceRegistryTokenUtility on gnosis mainnet (not needed if we do it via the initial deployment scripts)